src/libraries/System.Security.Principal.Windows/src/System/Security/Principal/WindowsPrincipal.cs

```diff
@@ -24,7 +24,7 @@ public enum WindowsBuiltInRole
 
     public class WindowsPrincipal : ClaimsPrincipal
     {
-        private readonly WindowsIdentity _identity = null;
+        private readonly WindowsIdentity _identity = null!;
```

Comment:
Where is the `private WindowsPrincipal() { }` constructor used?  That seems to be the only reason this `= null!` is needed, and I don't see anyone using it (plus the type has other ctors that will prevent a default public one from being added).

Reply:
It's dead code, but I opted for that approach instead of removing members.

Reply:
Let's remove the dead code.  Thanks.

Reply:
Done.